docs_user/modules/openstack-mariadb_copy.adoc

```diff
@@ -173,71 +173,66 @@ ____
+
+  # db server root password map
+  declare -A db_server_password_map
+  db_server_password_map['default']=${PODIFIED_DB_ROOT_PASSWORD}
+  db_server_password_map['nova_cell1']=${PODIFIED_DB_ROOT_PASSWORD}
+
+  cd /backup
+  for db_file in \$(ls *.sql); do
+    db_name=\$(echo \${db_file} | awk -F'.' '{ print \$1; }')
+    if [[ -v "db_name_map[\${db_name}]" ]]; then
+      echo "renaming \${db_name} to \${db_name_map[\${db_name}]}"
+      db_name=\${db_name_map[\${db_name}]}
     fi
-    db_server=${db_server_map["default"]}
-    if [[ -v "db_server_map[${db_name}]" ]]; then
-        db_server=${db_server_map[${db_name}]}
+    db_server=\${db_server_map["default"]}
+    if [[ -v "db_server_map[\${db_name}]" ]]; then
+      db_server=\${db_server_map[\${db_name}]}
     fi
-    db_password=${db_server_password_map["default"]}
-    if [[ -v "db_server_password_map[${db_name}]" ]]; then
-        db_password=${db_server_password_map[${db_name}]}
+    db_password=\${db_server_password_map['default']}
+    if [[ -v "db_server_password_map[\${db_name}]" ]]; then
+      db_password=\${db_server_password_map[\${db_name}]}
     fi
-    echo "creating ${db_name} in ${db_server}"
-    container_name=$(echo "mariadb-client-${db_name}-create" | sed 's/_/-/g')
-    sql_command="CREATE DATABASE IF NOT EXISTS ${db_name} DEFAULT CHARACTER SET ${CHARACTER_SET} DEFAULT COLLATE ${COLLATION};"
-    echo "SQL COMMAND: $sql_command"
-    oc run ${container_name} --image ${MARIADB_IMAGE} -i --rm --restart=Never -- \
-        mysql -h "${db_server}" -uroot "-p${db_password}" << EOF
-$sql_command
-EOF
-    echo "importing ${db_name} into ${db_server}"
-    container_name=$(echo "mariadb-client-${db_name}-restore" | sed 's/_/-/g')
-    oc run ${container_name} --image ${MARIADB_IMAGE} -i --rm --restart=Never -- \
-        mysql -h "${db_server}" -uroot "-p${db_password}" "${db_name}" < "${db_file}"
-done
-oc exec -it openstack-galera-0 -c galera -- mysql --user=root --password=${db_server_password_map["default"]} -e \
+    echo "creating \${db_name} in \${db_server}"
+    mysql -h"\${db_server}" -uroot "-p\${db_password}" -e \
+      "CREATE DATABASE IF NOT EXISTS \${db_name} DEFAULT \
+      CHARACTER SET ${CHARACTER_SET} DEFAULT COLLATE ${COLLATION};"
+    echo "importing \${db_name} into \${db_server}"
+    mysql -h "\${db_server}" -uroot "-p\${db_password}" "\${db_name}" < "\${db_file}"
+  done
+
+  mysql -h "\${db_server_map['default']}" -uroot -p"\${db_server_password_map['default']}" -e \
     "update nova_api.cell_mappings set name='cell1' where name='default';"
-oc exec -it openstack-cell1-galera-0 -c galera -- mysql --user=root --password=${db_server_password_map["default"]} -e \
+  mysql -h "\${db_server_map['nova_cell1']}" -uroot -p"\${db_server_password_map['nova_cell1']}" -e \
     "delete from nova_cell1.services where host not like '%nova-cell1-%' and services.binary != 'nova-compute';"
+EOF
 ----
 
 == Post-checks
```

Comment:
Github won't let me comment below, but there's still mention of the "ovn-data" pod in the docs below which should be replaced with "mariadb-copy-data".